src/FileMigrationHelper.php

```diff
@@ -56,27 +61,40 @@ public function run($base = null)
         Environment::increaseTimeLimitTo();
         Environment::increaseMemoryLimitTo();
 
+        $this->logger = Injector::inst()->get(LoggerInterface::class);
```

Comment:
I guess if we move this one to the constructor, then other methods of the class should be able to reuse the logger. Otherwise we don't have any benefits from keeping the logger as a property and could just hold in within a local variable.

Reply:
I would imagine, the constructor could be receiving a custom logger and go for the global one as a fallback if null passed. inversion of control ftw